code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm

```diff
@@ -999,8 +999,8 @@
 	adj_drowsy = -3
 	adj_temp = -5
 
-	glass_name = "Space Cola"
-	glass_desc = "A glass of refreshing Space Cola"
+	glass_name = "Cherry Cola"
+	glass_desc = "A glass of refreshing cherry-flavoured cola."
```

Comment:
```suggestion
	glass_desc = "A glass of refreshing cherry-flavoured Cola."
```

Reply:
More uppercase for cherry-flavoured Cola!